packaging/cockpit-starter-kit.spec.in

```diff
@@ -10,6 +10,8 @@ BuildArch: noarch
 BuildRequires:  nodejs
 BuildRequires: make
 BuildRequires: libappstream-glib
+BuildRequires: libappstream-glib-devel
```

Comment:
Why does it need that? I don't have it installed, and I can run the `msgfmt` call just fine.

That packae is not present in the mock environments, and we call the package build with `--offline`. This is why the CirrusCI test fails, and it will fail the same way on our CI.

Reply:
/usr/share/gettext/its/metainfo.{its,loc} are not present in the gettext package of RHEL/CentOS 8, and they are the needed metadata for gettext to know how to extract the messages from AppStream XML files, and inject translations back. libappstream-glib-devel provides those files on el8, so I can limit the BR only there if seems better.

Reply:
We'd still need to add that BR to our rhel-*, centos-8-stream, etc. images. Alternatively, the file could be prebuilt with `make dist`, so that the rpm build doesn't have to build it again. Then you need to split the Makefile rule, but that seems useful anyway?

Reply:
FTR: I'm fine with adding this BR to our mocks. [rhel-9-0 has been broken](https://github.com/cockpit-project/bots/pull/3027) for a while, but that shouldn't block this PR -- as we don't test starter-kit on RHEL anyway. So if you want to go ahead with this BR, let's do it.

Reply:
I don't have these files on my system either (I didn't install the -devel package), and the generated XML file looks quite reasonable.  In particular,

```xml
  <metadata_license>CC0-1.0</metadata_license>
  <name>Starter Kit</name>
  <name xml:lang="de">Bausatz</name>
```

Reply:
Correction: In Fedora 35, gettext ships them in /usr/share/gettext-0.21/its/metainfo.{its,loc}